examples/Basics/recommenders.py

```diff
@@ -84,7 +84,7 @@
 #   - If a larger value is chosen, only those that allow batching.
 #       That is, 'q'-variants of the acquisition functions must be chosen.
 
-# The default he acquisition function is q-Expected Improvement.
+# The default acquisition function is q-Expected Improvement.
 
 ACQ_FUNCTION = "qEI"
```

Comment:
since creation of this example the global acqf default has been changed to `qLogEI` so this PR is an opportunity to change that in this example as well

Reply:
@Hrovatin could you also just quickly add that in? Tehn I'll approve and we can merge.

Reply:
the sentence above should also be amended to say its log

what do you mean by `missed` ?

Reply:
2.) `missed` - things that are in baybe but not mentioned here - it took me a bit longer than needed to find the UCB and how to use it for example as it is only explained in reinforcement learning tutorial.

Reply:
yes there are several aspects snot well documetned yet, like ACQF functions. I dont think this necessarily means that it needs to be sued in this recommender user guide, instead we should have a proper acqf user guide at some point